package.json

```diff
@@ -74,7 +74,7 @@
     "jspdf": "https://github.com/parallax/jsPDF.git#b7a1d8239c596292ce86dafa77f05987bcfa2e6e",
     "jszip": "^3.10.1",
     "kolibri-constants": "^0.2.0",
-    "kolibri-design-system": "^4.5.1",
+    "kolibri-design-system": "5.0.0-rc7",
```

Comment:
Hi! Just jumping in to notice that there have been several breaking changes in KDS v5, so these will probably need to be addressed for this update. (whether is before or after this PR).

Reply:
Super! Thanks @akolson n.n